src/credentials.js

```diff
@@ -73,7 +73,7 @@ class Credentials extends EventEmitter {
       }
     });
   }
-  login(captcha = {}) {
+  login(captcha = {}, options = {}) {
```

Comment:
The code must provide proper abstraction for this - using `options` is very unclear and confusing. Or at least provide a documentation for fields.